srcpkgs/gnome-core/template

```diff
@@ -9,44 +9,45 @@ license="GPL-2.0-only, LGPL-2.0-only"
 homepage="https://www.gnome.org"
 
 depends="
- adwaita-icon-theme>=41.0
- at-spi2-core>=2.42.0
+ adwaita-icon-theme>=42.0
+ at-spi2-core>=2.44.0
  at-spi2-atk>=2.38.0
  cantarell-fonts>=0.301
  dconf>=0.40.0
  evolution-data-server>=3.42.0
- folks>=0.14.0
+ folks>=0.15.4
  font-adobe-source-code-pro>=2.030R
- gcr>=3.40.0
- gdm>=41.0
+ gcr>=3.41.0
+ gdm>=42.0
  geocode-glib>=3.26.2
  gfbgraph>=0.2.4
- gjs>=1.70.0
- glib>=2.70.0
- glib-networking>=2.70.0
+ gjs>=1.72.0
+ glib>=2.72.0
+ glib-networking>=2.72.0
```

Comment:
This would break the package. There wasn't a PR for glib-networking 2.72 and it's not included in this PR.

Reply:
Thanks for pointing that out.